src/main.rs

```diff
@@ -32,7 +32,13 @@ mod version;
 fn main() {
     env_logger::init().unwrap_or_else(|e| panic!("{}", e));
 
-    let cfg = config::Config::new();
+    let cfg = match config::Config::new() {
+        Ok(c) => c,
+        Err(e) => {
+            println!("{}", e);
```

Comment:
I think I'd like this to say "Failed to read config {}", because currently the error message still is somewhat cryptic.